var/spack/repos/builtin/packages/mpifileutils/package.py

```diff
@@ -102,6 +106,12 @@ def cmake_args(self):
         else:
             args.append("-DENABLE_EXPERIMENTAL=OFF")
 
+        if "+daos" in self.spec:
+            args.append("-DENABLE_DAOS=ON")
+            args.append("-DWITH_DAOS_PREFIX=%s" % self.spec["daos"].prefix)
+        else:
+            args.append("-DENABLE_DAOS=OFF")
```

Comment:
Also,  more as  an FYI in case you weren't aware, CMakePackage has some handy helper methods like `self.define_from_variant()`, described at https://spack.readthedocs.io/en/latest/build_systems/cmakepackage.html#adding-flags-to-cmake.

Reply:
I did it this way because it is consistent with the rest of the package, I generally try to stick with whatever the maintainer was using.